server/svix-server/src/v1/endpoints/retry_schedule.rs

```diff
@@ -0,0 +1,75 @@
+// SPDX-FileCopyrightText: © 2022 Svix Authors
```

Comment:
I'd put these in the same file as the other event_type stuff, just at the bottom. Or at the very least name it event_type_retry_schedule. Though this doesn't conform with the rest of our organization, so putting it in the other file makes more sense.